### PR TITLE
Disable widgets interactivity

### LINKED ISSUE #284: Add disableInteractivity flag for widgets
It would be nice to have an option in the widgets to disable interactivity. This should consist of disabling pointer events and hiding the footer.

This is the CSS I'm using for the workaround:

```css
.widget {
  pointer-events: none;
  footer {
    display: none ;
  }
}
```

### PR DESCRIPTION
**Pull request by jesusbotella**
Fixes #284 and #209

### CODE REVIEW

File: packages/components/src/components/as-category-widget/as-category-widget.tsx [hunk 1]
```diff
@@ -133,7 +136,7 @@ export class CategoryWidget {
     return [
       this._renderHeader(),
       this._renderCategoryList(),
-      this._renderFooter()
+      !this.disableInteractivity ? this._renderFooter() : ''
```
[the remark this answers is not in the document]
Comment (jesusbotella) [reply packages/components/src/components/as-category-widget/as-category-widget.tsx]: I don't like the empty string in first place 😁

[the remark this answers is not in the document]
Comment (IagoLast) [reply packages/components/src/components/as-category-widget/as-category-widget.tsx]: I dont like stupid double negations

[the remark this answers is not in the document]
Comment (IagoLast) [reply packages/components/src/components/as-category-widget/as-category-widget.tsx]: invoking @ivanmalagon to resolve this battle

[the remark this answers is not in the document]
Comment (IagoLast) [reply packages/components/src/components/as-category-widget/as-category-widget.tsx]: :trollface:

[the remark this answers is not in the document]
Comment (IagoLast) [reply packages/components/src/components/as-category-widget/as-category-widget.tsx]: 🌮

**Pull request merged**